lib/internal/repl.js

```diff
@@ -165,3 +156,60 @@ function _replHistoryMessage() {
   this._historyPrev = Interface.prototype._historyPrev;
   return this._historyPrev();
 }
+
+
+function getTMPFile(ready) {
+  var tmpPath = os.tmpdir();
+  if (!tmpPath) {
+    return ready(new Error('no tmpdir available'));
+  }
+  tmpPath = path.join(tmpPath, process.pid + '-' + 'repl.tmp');
```

Comment:
Maybe `'node-repl-' + process.pid` to make it clear which process created that file?